apps/public-reference/pages/applications/review/demographics.tsx

```diff
@@ -42,29 +76,97 @@ export default () => {
       </FormCard>
 
       <FormCard>
-        <p className="text-bold">
+        <p className="form-card__back">
           <strong>
-            <Link href="/applications/preferences/general">
+            <Link href="/applications/contact/name">
```

Comment:
I think this is the wrong URL for linking back…

Reply:
oh, you're right, I'll fix it! I'm also thinking about these hardcoded links. Maybe will be better to create something like "routes" constant & helper function to detect prev/next component (not necessary, but helpful). What do you think?

Reply:
@dominikx96 Yeah, we have a project underway to consolidate all the form routes into a single config file, see #453 (and feedback is most welcome)